<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NGSDReplicationWidget</class>
 <widget class="QWidget" name="NGSDReplicationWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>3</number>
   </property>
   <property name="leftMargin">
    <number>3</number>
   </property>
   <property name="topMargin">
    <number>3</number>
   </property>
   <property name="rightMargin">
    <number>3</number>
   </property>
   <property name="bottomMargin">
    <number>3</number>
   </property>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Converts information from HG19 NGSD to HG38 NGSD.&lt;br/&gt;&lt;br/&gt;To port your HG19 NGSD to HG38, perform the following steps:&lt;br/&gt;1) add the HG38 genome and NGSD credentials to the settings.ini file: reference_genome_hg38, ngsd_hg38_host, ngsd_hg38_port ...&lt;br/&gt;2) replicate base data only using this dialog&lt;br/&gt;3) update processing system table (target_file, genome_id) for all processing systems that will be re-analyzed based on HG38 (make sure the genome is called 'GRCh38').&lt;br/&gt;4) set up HG38 megSAP and re-analyze the samples with it (ma,vc,cn,sv,db)&lt;br/&gt;5) re-do CNV calling as it is cohort based (cn,db)&lt;br/&gt;6) replicate variant data using this dialog&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Actions</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="leftMargin">
       <number>3</number>
      </property>
      <property name="topMargin">
       <number>3</number>
      </property>
      <property name="rightMargin">
       <number>3</number>
      </property>
      <property name="bottomMargin">
       <number>3</number>
      </property>
      <property name="spacing">
       <number>3</number>
      </property>
      <item row="4" column="0">
       <widget class="QCheckBox" name="pre_checks">
        <property name="text">
         <string>perform pre-replication checks</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QCheckBox" name="post_checks">
        <property name="text">
         <string>perform post-replication checks</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>3</number>
        </property>
        <item>
         <widget class="QCheckBox" name="variant_data">
          <property name="text">
           <string>replicate variant data</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="skip_rc">
          <property name="text">
           <string>skip report config (otherwise re-analysis of samples is not possible)</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="5" column="0">
       <widget class="QCheckBox" name="base_data">
        <property name="text">
         <string>replicate base data</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="repliate_btn">
     <property name="text">
      <string>replicate</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextBrowser" name="output">
     <property name="lineWrapMode">
      <enum>QTextEdit::NoWrap</enum>
     </property>
     <property name="openExternalLinks">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
